samples/excel/85-preview-apis/pivottable-slicer.yaml

```diff
@@ -0,0 +1,211 @@
+                const sheetPivot = await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Pivot");
+
+                const data = [
+                    ["Farm", "Type", "Classification", "Crates Sold at Farm", "Crates Sold Wholesale"],
+                    ["A Farms", "Lime", "Organic", 300, 2000],
+                    ["A Farms", "Lemon", "Organic", 250, 1800],
+                    ["A Farms", "Orange", "Organic", 200, 2200],
+                    ["B Farms", "Lime", "Conventional", 80, 1000],
+                    ["B Farms", "Lemon", "Conventional", 75, 1230],
+                    ["B Farms", "Orange", "Conventional", 25, 800],
+                    ["B Farms", "Orange", "Organic", 20, 500],
+                    ["B Farms", "Lemon", "Organic", 10, 770],
+                    ["B Farms", "Kiwi", "Conventional", 30, 300],
+                    ["B Farms", "Lime", "Organic", 50, 400],
+                    ["C Farms", "Apple", "Organic", 275, 220],
+                    ["C Farms", "Kiwi", "Organic", 200, 120],
+                    ["D Farms", "Apple", "Conventional", 100, 3000],
+                    ["D Farms", "Apple", "Organic", 80, 2800],
+                    ["E Farms", "Lime", "Conventional", 160, 2700],
+                    ["E Farms", "Orange", "Conventional", 180, 2000],
+                    ["E Farms", "Apple", "Conventional", 245, 2200],
+                    ["E Farms", "Kiwi", "Conventional", 200, 1500],
+                    ["F Farms", "Kiwi", "Organic", 100, 150],
+                    ["F Farms", "Lemon", "Conventional", 150, 270]
+                ];
+
+                const range = sheetData.getRange("A1:E21");
+                range.values = data;
+                range.format.autofitColumns();
+
+                sheetPivot.activate();
+
+                await context.sync();
+            });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            } catch (error) {
+                // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+                console.error(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to work with a slicer on a PivotTable.</p>
+        </section>
+
+        <section class="samples ms-font-m">
+          <h3>Setup</h3>
+          <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Add data</span>
+            </button>
+          <p />
+          <button id="add-pivot-table" class="ms-Button">
+            <span class="ms-Button-label">Add pivot table</span>
+          </button>
+        </section>
+        <section class="samples ms-font-m">
+          <h3>Try it out</h3>  
```

Comment:
If there's an order in which the buttons in the Try it Out section should be pushed, might want to add an instruction about that.